includes/rest/class-actors-controller.php

```diff
@@ -101,23 +101,15 @@ public function get_item( $request ) {
 			return $user;
 		}
 
-		$link_header = \sprintf( '<%1$s>; rel="alternate"; type="application/activity+json"', $user->get_id() );
-
-		// Redirect to canonical URL if it is not an ActivityPub request.
-		if ( ! is_activitypub_request() ) {
-			\header( 'Link: ' . $link_header );
-			\header( 'Location: ' . $user->get_canonical_url(), true, 301 );
-			exit;
-		}
-
 		/**
 		 * Action triggered prior to the ActivityPub profile being created and sent to the client.
 		 */
 		\do_action( 'activitypub_rest_users_pre' );
 
 		$data = $user->to_array();
 
-		$response = \rest_ensure_response( $data );
+		$response    = \rest_ensure_response( $data );
+		$link_header = \sprintf( '<%1$s>; rel="alternate"; type="application/activity+json"', $user->get_id() );
 		$response->header( 'Content-Type', 'application/activity+json; charset=' . \get_option( 'blog_charset' ) );
 		$response->header( 'Link', $link_header );
```

Comment:
```suggestion
		$response = \rest_ensure_response( $data );
		$response->header( 'Content-Type', 'application/activity+json; charset=' . \get_option( 'blog_charset' ) );
		$response->header( 'Link', \sprintf( '<%1$s>; rel="alternate"; type="application/activity+json"', $user->get_id() ) );
```
Can be moved in line since it's not used anywhere else